### PR TITLE
feat(DSC): add a resource to manage obs Asset

### PR DESCRIPTION
**Pull request by hmbig2**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [X] Tests added/passed.
* [X] Documentation updated.
* [X] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/dsc' TESTARGS='-run=TestAccAssertObs_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/dsc -v -run=TestAccAssertObs_basic -timeout 360m -parallel 4
=== RUN   TestAccAssertObs_basic
=== PAUSE TestAccAssertObs_basic
=== CONT  TestAccAssertObs_basic
--- PASS: TestAccAssertObs_basic (75.28s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dsc       75.323s
```

### CODE REVIEW

File: docs/resources/dsc_assert_obs.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+subcategory: "Data Security Center (DSC)"
+---
+
+# huaweicloud_dsc_assert_obs
+
+Manages a OBS assert resource of DSC within HuaweiCloud.
```
Comment (Lance52259) [inline docs/resources/dsc_assert_obs.md]: s/a/an

File: huaweicloud/services/dsc/resource_huaweicloud_dsc_assert_obs.go [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+			},
+		},
+	}
+}
+
+func resourceAssertObsCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	config := meta.(*config.Config)
+	region := config.GetRegion(d)
+
+	// createAssertObs: create a OBS assert .
```
Comment (Lance52259) [inline huaweicloud/services/dsc/resource_huaweicloud_dsc_assert_obs.go]: ditto.

File: docs/resources/dsc_assert_obs.md (hunk 1 above)
Comment (hmbig2) [reply docs/resources/dsc_assert_obs.md]: done

File: huaweicloud/services/dsc/resource_huaweicloud_dsc_assert_obs.go (hunk 2 above)
Comment (hmbig2) [reply huaweicloud/services/dsc/resource_huaweicloud_dsc_assert_obs.go]: done

File: docs/resources/dsc_assert_obs.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+---
+subcategory: "Data Security Center (DSC)"
+---
+
+# huaweicloud_dsc_assert_obs
```
Comment (Jason-Zhang9309) [inline docs/resources/dsc_assert_obs.md]: should be asset?

Comment (Lance52259) [reply docs/resources/dsc_assert_obs.md]: Seems right

Comment (hmbig2) [reply docs/resources/dsc_assert_obs.md]: done

**Pull request merged**